pkg/eth/rpc_types.go

```diff
@@ -67,6 +70,15 @@ func (t *SendTransactionRequest) GasPriceHex() string {
 	return t.GasPrice.Hex()
 }
 
+// ========== eth_sendRawTransaction ============= //
+
+type (
+	// Presents hexed string of a raw transaction
+	SendRawTransactionRequest [1]string
+	// Presents hexed string of a transaction hash
+	SendRawTransactionResponse [1]string
+)
```

Comment:
Unnecessary as sendRawTransaction has already been implemented. See eth_sendRawTransaciton under the transformer package.